packages/kit/src/runtime/server/respond.js

```diff
@@ -607,7 +607,14 @@ export async function respond(request, options, manifest, state) {
 
 			// we can't load the endpoint from our own manifest,
 			// so we need to make an actual HTTP request
-			return await fetch(request);
+			const fetchResponse = await fetch(request);
+
+			// the header for the response needs to be mutable, so we need to clone it
```

Comment:
can you expand the comment to explain why they need to be mutated?
```suggestion
			// the headers for the response needs to be mutable, so we need to clone it
```